src/Lynx/Form.purs

```diff
@@ -49,38 +49,58 @@ type Section f
     , fields :: NonEmpty Array (Field f)
     }
 
+type SequenceSection f
+  = { id :: String
+    , name :: String
+    , fields :: NonEmpty Array (Field f)
+    }
+
 type Sequence f
   = { name :: String
     , key :: String
     , template :: TemplateSection f
-    , values :: InputSource (Array (Section f))
+    , values :: InputSource (Array (SequenceSection f))
     }
 
-stamp :: forall a. Key -> Page a -> Page a
-stamp key page = page { tabs = map stampTab page.tabs }
+stamp :: forall a f. Monad f => f String -> Key -> Page a -> f (Page a)
```

Comment:
I think we should agree on consistent type variables to avoid confusion. Throughout this file we use `f` to mean either `Expr` or `ExprType`. It's carried over from when our expressions were typed and `f` was the Functor wrapping our types `Expr a` vs `Identity a`. I don't know if we want to keep using `f` or rename it to something else, I've noticed `a` used instead in discussions and elsewhere.

Also in `src/Lynx.purs` we use `m` for our Monad. I'm okay with changing things, but I'd like to keep things consistent if we do.